src/utils.js

```diff
@@ -0,0 +1,4 @@
+export const isTouch = () => {
+  // TODO - add IE support
+  return 'ontouchstart' in window
+}
```

Comment:
This doesn't need to be a function, can be just a constant, as this is not going to change after the page load.

Reply:
Good call, will update in upcoming commit along with making sure it works for all browsers. Currently only working in Firefox. Might need to add something like:

```
if (('ontouchstart' in window) ||
     (navigator.maxTouchPoints > 0) ||
     (navigator.msMaxTouchPoints > 0)) {
      /* browser with either Touch Events of Pointer Events
         running on touch-capable device */
}
```
https://hacks.mozilla.org/2013/04/detecting-touch-its-the-why-not-the-how/